config/module.go

```diff
@@ -234,34 +232,44 @@ func loadValidators(k *KoanfAdapter, exportedConfigs []ExportedConfig) error {
 func getHandler(style string) (handler, error) {
 	switch style {
 	case "json":
-		return jsonHandler{}, nil
+		return rewriteHandler{codec: json.NewCodec(json.WithIndent("  "))}, nil
 	case "yaml":
-		return yamlHandler{}, nil
+		return appendHandler{codec: yaml.Codec{}}, nil
```

Comment:
Why are the two handlers named differently?

Reply:
I think so:

The unmarshaler was injected. 
There is no need to declare which unmarshaler to use. 
Just state their differences: append or rewrite.

Reply:
oh, should be, my mistake.